app/javascript/components/set-ownership-form/index.jsx

```diff
@@ -8,26 +8,58 @@ import createSchema from './ownership-form.schema';
 class SetServiceOwnershipForm extends Component {
```

Comment:
You don't actually rename the component, its name is the same. Also I suggest you to do the renaming first in a separate PR.